README.md

```diff
@@ -12,6 +12,8 @@
 
 ## Statistical Tools (Widgets)
 
+- [GitClear Commit Activity Bubble Graph](https://www.gitclear.com/github_profile_dynamic_readme_free#commit_activity) - Dynamically generating daily visualization of which tickets/repos/branches are being worked on by individual or team
+- [GitClear Area Charts for GitHub Profiles](https://www.gitclear.com/github_profile_dynamic_readme_free#area_graph) - Dynamically generating area charts to showcase relative velocity of work across repos
```

Comment:
Instead of [GitClear Area Charts for GitHub Profiles] rename it to [GitClear Area Charts].
Same here.